crates/ra_hir_def/src/path.rs

```diff
@@ -78,6 +78,12 @@ impl ModPath {
         }
         self.segments.first()
     }
+
+    pub fn to_ast(&self) -> ast::Path {
+        use ast::AstNode;
+        let parse = ast::SourceFile::parse(&self.to_string());
+        parse.tree().syntax().descendants().find_map(ast::Path::cast).unwrap()
+    }
```

Comment:
Having this API here feels wrong, as it is fundamentally lossy and erases hygiene. Let's move it to `ra_assists util.

Rendering to string is also lossy, of course, but it's obvious enough that you can't imagine that it does the right thing